.gitignore

```diff
@@ -6,6 +6,7 @@
 /tmp
 /out-tsc
 /bazel-out
+/src/assets/config 
```

Comment:
You should probably just ignore the one file, so we can put a template file in here for Cloud to work from.